Cargo.toml

```diff
@@ -34,7 +34,9 @@ default = ["std"]
 # Enables `once_cell::sync` module.
 std = ["alloc"]
 # Enables `once_cell::race::OnceBox` type.
-alloc = []
+alloc = ["race"]
+# Enables `once_cell::race` module.
+race = []
```

Comment:
This should be added to `default` above I think?

Reply:
Oh, it's not transitive (`std` depends on `alloc` with depends on `race`)? Hmmm... anyway CI fails, so I better fix that :P.

Reply:
Ah, you are of course right!